pkg/filehelpers/file_helper.go

```diff
@@ -0,0 +1,66 @@
+package filehelpers
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+func CopyDir(src string, dst string) error {
```

Comment:
Where is this used?

Reply:
Just git fetch?

Reply:
yes, the copy file is used in build init as well, but copy dir is only used in git

Reply:
I ended up getting rid of the file helper since it wasn't really necessary, I just put the function in the two places it was needed